core/lib/dal/src/models/storage_eth_tx.rs

```diff
@@ -22,6 +22,7 @@ pub struct StorageEthTx {
     pub updated_at: NaiveDateTime,
     // TODO (SMA-1614): remove the field
     pub sent_at_block: Option<i32>,
+    pub from_addr: Option<Vec<u8>>,
```

Comment:
please add comment 
(and what does it mean if it is not set.)

Also - maybe the type should be address? rather than u8?